install.sh

```diff
@@ -14,6 +14,9 @@ ZONEINFO_DIR="/usr/share/zoneinfo/"
 
 uninstall () {
     rm -rf ${CACHE_DIR}*
+
+    rm -rf /etc/systemd/system/xochitl.service.d
```

Comment:
This should remove the file you created, and the directory only if it's empty. There may be other override files that this is removing.

Reply:
Agree, should be fixed in the last commit.